source/exceptions.tex

```diff
@@ -1016,9 +1016,9 @@
 
 \pnum
 \indextext{\idxcode{terminate}}%
-% FIXME: What does it mean to abandon exception handling?
-In some situations, exception handling is abandoned
-for less subtle error handling techniques.
+Some errors in a program cannot be recovered from, such as when an exception is
+not handled or a thread is destroyed while still running. In such cases,
```

Comment:
What does it mean to "destroy a thread"?

Reply:
Updated to refer to a `std::thread` object.  Force-pushed again to support a clean merge.